### PR TITLE
Dockerize backend and auto add fixtures

### PR DESCRIPTION
**Pull request by edalholt**
Closing #147 

Implementing:
`docker compose up`
This will create a local mongoDB and start backend

Also updates the backend-start scripts for using docker

### CODE REVIEW

File: backend/.env.example [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+MONGO_INITDB_ROOT_USERNAME="opptakDev"
+MONGO_INITDB_ROOT_PASSWORD="topSecret"
+
+## Optional development variables
+FRONTEND_URI = "http://localhost:3000"
+API_URI = "https://dev.api.ntnui.no"
+MAIN_BOARD_ID = 9
+ELECTION_COMMITTEE_ID = 10
+
+NTNUI_TOOLS_API_URL = 'https://dev.api.ntnui.no'
```
Comment (edalholt) [inline backend/.env.example:12]: This does not work, requests are still sent to https://api.ntnui.no/

File: backend/Dockerfile [hunk 2]
```diff
@@ -9,8 +9,5 @@ COPY tsconfig.json ./
 RUN npm install
 COPY . .
 RUN npm run build
-ENV PORT=8080
-ENV DB_URI=${DB_URI}
```
Comment (edalholt) [inline backend/Dockerfile:13]: Maybe we should add a build stage for production if these lines are necessary?

Comment (Xtrah) [reply backend/Dockerfile:13]: Whatever is more elegant and understandable 🤓

File: backend/README.md [hunk 3]
```diff
@@ -9,31 +9,18 @@
 
 ## Developement
 
-Install and run the backend using `npm install`, then `npm run app`, or simply run `npm run backend` from the project root folder.
+Install and run the backend using `docker compose up`  
+If you change the fixtures afterwards you have to run `docker compose up --build` to reload the new fixtures to the image.
 
 ### Environment variables
```
Comment (Xtrah) [inline backend/README.md:15]: Is it possible to run `docker compose up` without the `.env` file? If not, we should rearrange the README so requirements comes before starting the app. 😊

Also, is there any situation where one wouldn't want to run using Docker? As in, any reason to keep both installation instructions?

File: backend/README.md [hunk 4]
```diff
@@ -9,31 +9,18 @@
...
-API_URI = "https://dev.api.ntnui.no"
-MAIN_BOARD_ID = 9
-ELECTION_COMMITTEE_ID = 10
-
-## Optional ntnui-tools development variable
-NTNUI_TOOLS_API_URL = "https://dev.api.ntnui.no"
-
-```
+Rename `.env.example` to `.env`  
+When using docker you don't need to do any modifications
```
Comment (Xtrah) [inline backend/README.md:18]: A suggestion to make it more beginner friendly <3

```suggestion
Environment variables are injected automatically from an `.env` file.

The `.env.example` file is set up with default variables to use during development locally and shouldn't need any modifications for first time use. Simply rename `.env.example` to `.env` 🏃
```

**Review by Xtrah — APPROVED** [review]
Works like a charm! A few minor suggestions, otherwise a great improvement in set up time and ease of use! 👏📈

**Comment by ivarhe** [comment]
Docker is the way 🐳 

mando-way-this-is-the-way

**Pull request merged**